### ISSUE TITLE
Cannot install on OSX

### ISSUE THREAD

**Issue by 4rch3r23** [Velocidex/velociraptor#33]
Maybe I’m missing something to install on OSX.
I downloaded the Darwin release and ran `“releaseAmd64 —config client.config.yaml service install”`
The client did not enroll in to the fleet.
The service ran and there weren’t any errors that were displayed.
What am I missing here?
Also, what happened to the google group?

**Comment by scudette** [comment]
You are right - the darwin installer was not updated to copy the config file separately (which is the way it is done in Windows now).   In the past we relied on the config being embedded inside the binary:

```
$ velociraptor config repack client.config.yaml my_velociraptor_darwin.mach 
```

This approach is left over from GRR which also embeds the config but the problem is that this prevents us from distributing signed binaries (because the config invalidates the signature). We changed the windows binary to copy the config separately and so not modify the actual binary.

Packing the config inside the binary will be deprecated (although it is currently still possible we dont actually document it). But it is currently necessary for osx.

The mailing list is still velociraptor-discuss@googlegroups.com  - does this not work?
